apps/Caskaydia Cove NF/uninstall

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-rm -r ~/.fonts/Casaydia
+sudo rm -r /usr/local/share/fonts/Caskaydia
```

Comment:
-rf so that a failure due to the folder already not existing does not cause the script to exit with an error